packages/jetpack-ai-calypso/src/logo-generator/components/generator-modal.scss

```diff
@@ -1,18 +1,24 @@
 .jetpack-ai-logo-generator-modal {
 	.components-modal__header {
-		border-bottom: 1px solid var(--studio-gray-5);
+		border-bottom: 1px solid var(--studio-gray-5, #dcdcde);
 		padding: 20px 24px;
 	}
 
 	.components-modal__content {
 		padding: 32px;
+
+		> div:not(.components-modal__header) {
+			height: 100%;
+		}
 	}
 
 	&__body {
```

Comment:
Not part of this change, but same comment: let's use full strings so they are easy to find/spot